### PR TITLE
HW1 Frameworks

### PR DESCRIPTION
**Pull request by sancho-sumy**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const port = 3000;
+
+const app = express();
+
+app.use(express.json());
+
+app.listen(port, () => {
+	logger.info(`App listening on http://localhost:${port}`);
+
+	app.use('/movies', moviesRoutes);
```
Comment (VitalyaIlin) [inline src/app.js]: It's not necessary to place lines 17-27 inside this event handler. Here you need just print message to make sure that app is running. Move lines 17-27 below the middleware in line 12

File: src/routes/health.routes.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ *       200:
+ *         description: App is up and running
+ *         content:
+ *           application/json:
+ *             example:
+ *               "message": "App is up and running"
+ *               "result": "OK"
+ */
+router.get('/', getStatusHandler);
+
```
Comment (VitalyaIlin) [inline src/routes/health.routes.js:25]: Optional: Add one more endpoint to fetch movie by ID and handle 404 error if movie is not found

File: src/controllers/health.controllers.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export async function getStatusHandler(req, res, next) {
+	try {
+		res.status(200).json({ message: 'App is up and running', result: 'OK' });
+	} catch (error) {
+		if (!error.statusCode) {
```
Comment (VitalyaIlin) [inline src/controllers/health.controllers.js]: you already implement the same logic inside error handling middleware

File: src/app.js (hunk 1 above)
Comment (sancho-sumy) [reply src/app.js]: Done!

File: src/routes/health.routes.js (hunk 2 above)
Comment (sancho-sumy) [reply src/routes/health.routes.js:25]: Done!

File: src/controllers/health.controllers.js (hunk 3 above)
Comment (sancho-sumy) [reply src/controllers/health.controllers.js]: Done!

**Review by VitalyaIlin — APPROVED** [review]

**Pull request merged**